app/models/setting/rh_cloud.rb

```diff
@@ -5,6 +5,7 @@ def self.default_settings
     [
       set('allow_auto_inventory_upload', N_('Allow automatic upload of the host inventory to the Red Hat cloud'), true),
       set('obfuscate_inventory_hostnames', N_('Obfuscate host names sent to Red Hat cloud'), false),
+      set('rh_cloud_token', N_('Authentication token to RH cloud services. Used to authenticate requests to Red Hat cloud APIs'), 'DEFAULT', N_('RH Cloud token'), nil, encrypted: true),
```

Comment:
There's `RH cloud`, `Red Hat cloud` and `RH Cloud` on this line. This should probably be unified